src/wagtail_personalisation/models.py

```diff
@@ -101,9 +93,9 @@ def get_rules(self):
         return segment_rules
 
     def toggle(self, save=True):
-        self.status = (
-            self.STATUS_ENABLED if self.status == self.STATUS_DISABLED
-            else self.STATUS_DISABLED)
+        self.enabled = (
```

Comment:
Might be able to shorten this to `self.enabled = not self.enabled` but not sure if that is good practice.